py/torch_migraphx/fx/converters/acc_ops_converters.py

```diff
@@ -109,6 +259,74 @@ def acc_ops_linear(mgx_module, node, args, kwargs):
     return MGXInstruction(out_mgx)
 
 
+# NLL (negative log likelihood loss) converter.  This op assumes data has already been normalized with log_softmax
+# to give meaningful results, although there is no restriction on input values.
+@migraphx_converter(acc_ops.nll_loss_forward)
+def acc_ops_nll_loss_forward(mgx_module, node, args, kwargs):
```

Comment:
```suggestion
@migraphx_converter(acc_ops.nll_loss)
def acc_ops_nll_loss(mgx_module, node, args, kwargs):
```
Necessary based on other name change requested